packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -535,14 +537,21 @@ function getFormBodyAndHeaders(
 	return { body: postBody, headers: header };
 }
 
-export function evalBlobsAndTrees(snapshot: ISnapshotContents) {
-	const trees = countTreesInSnapshotTree(snapshot.snapshotTree);
-	const numBlobs = snapshot.blobs.size;
+export function evalBlobsAndTrees(snapshot: ISnapshotContents | IPartialSnapshotWithContents) {
+	let trees: number;
+	let blobs: Map<string, ArrayBuffer>;
+	if (isFullSnapshot(snapshot)) {
+		trees = countTreesInSnapshotTree(snapshot.snapshotTree);
```

Comment:
See comment above - feels like this code should not change (if we change ISnapshotContents)